modules/migration-state-migration-cli.adoc

```diff
@@ -18,3 +19,39 @@ You can perform a state migration between clusters or within the same cluster.
 ====
 State migration migrates only the components that constitute an application's state. If you want to migrate an entire namespace, use stage or cutover migration.
 ====
+
+.Prerequisites
+
+* The application on the source cluster has its state persisted in `PersistentVolumes` provisioned through `PersistentVolumeClaims`.
```

Comment:
You could update this prerequisite to use active voice. For example:

> * The state for the application on the source cluster is persisted in `PersistentVolumes` provisioned through `PersistentVolumeClaims`.

Reply:
@pneedle-rh  I've made this change but it's still passive voice.